packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.ts

```diff
@@ -984,6 +984,12 @@ export const defaultConfigs: { [name: string]: { type: string; [key: string]: an
     bridgeAdminAddress: "0x30B44C676A05F1264d1dE9cC31dB5F2A945186b6",
     rateModels: across.constants.RATE_MODELS,
   },
+  uSPAC10: {
+    type: "uSPAC",
+    lookback: 7200,
+    correctionFactor: "1.0",
```

Comment:
What is `correctionFactor` parameter? There is no reference to this in the draft UMIP

Reply:
We have added information on `correctionFactor` to UMIP